apps/web/src/styles/theme.ts

```diff
@@ -117,13 +126,59 @@ const theme = extendTheme({
     },
     Button: {
       variants: {
-        solid: {
-          bg: "var(--chakra-colors-gray-100)",
+        solidPrimary: {
```

Comment:
I think it's better to stick to the names we have in figma: primary, secondary, tertiary